src/main/java/api/salesforce/Authentication.java

```diff
@@ -1,11 +1,11 @@
 package api.salesforce;
 
+import static io.restassured.RestAssured.given;
+
```

Comment:
Delete line too

Reply:
It is required by checkstyle for static imports.